core/modules/replica/ApplicationTypes.h

```diff
@@ -1,6 +1,5 @@
 /*
  * LSST Data Management System
- * Copyright 2017 LSST Corporation.
  *
```

Comment:
I think you should check with KT (or someone familiar with the copyright rules) to make sure this meets up with LSST  copyright requirements in all cases.

Reply:
KT has already posted notes on tis subject last Monday. So, the copyright section has been removed from files as per the following guidelines:
https://developer.lsst.io/stack/license-and-copyright.html